array-methods.js

```diff
@@ -58,20 +75,48 @@ var stateSums = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var sumOfHighInterests = null;
-
+var sumOfHighInterests = dataset.bankBalances
+  .filter(c => !theseStates.includes(c.state))
+  .reduce((a, c) => {
+    if (!a.includes(c.state)) {
+      a.push(c.state);
+    }
+    return a;
+  }, [])
+  .reduce((a, c) => {
+    let stateInt = Math.round(stateSums[c] * 0.189);
+    if (stateInt > 50000) {
+      a += stateInt;
+    }
+    return a;
+  }, 0);
```

Comment:
This should be using stateSums instead of trying to recalculate the sums on the fly.